tests/foreman/longrun/test_inc_updates.py

```diff
@@ -34,6 +34,7 @@ class IncrementalUpdateTestCase(TestCase):
     def setUpClass(cls):
         """Creates all the pre-requisites for the Incremental updates test"""
         super(IncrementalUpdateTestCase, cls).setUpClass()
+        cls.old_task_timeout = entity_mixins.TASK_TIMEOUT
```

Comment:
Yes I thought about this.  But cls.old_task_timeout is now used in two places.  So it is better to capture the default value at the start of the test and reset to this value whenever it is modified in the course of the test.